pkg/identity/cache/test_utils.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cache
+
+import (
+	"time"
+
+	"github.com/cilium/cilium/pkg/identity"
+	"github.com/cilium/cilium/pkg/inctimer"
+	"github.com/cilium/cilium/pkg/labels"
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+type DummyOwner struct {
+	updated chan identity.NumericIdentity
+	mutex   lock.Mutex
+	cache   IdentityCache
+}
+
+func NewDummyOwner() *DummyOwner {
```

Comment:
Do you want to export all those functions?

Reply:
Yes, the interface requires them. A function that creates some manager/controller objects require such interface, where DummyOwner is used for in tests.